testing/hexer/APKBUILD

```diff
@@ -0,0 +1,42 @@
+# Contributor: Bradley J Chambers <brad.chambers@gmail.com>
+# Maintainer: Bradley J Chambers <brad.chambers@gmail.com>
+pkgname=hexer
+pkgver=1.4.0
+pkgrel=0
+pkgdesc="LAS and OGR hexagonal density and boundary surface generation"
+url="https://github.com/hobu/hexer"
+arch="all"
+license="LGPL2+"
+options="!check"
+depends="libgcc libstdc++ musl gdal"
+makedepends="cmake gdal-dev"
+install=""
+subpackages="$pkgname-dev"
+source="$pkgname-$pkgver.tar.gz::https://github.com/hobu/$pkgname/archive/$pkgver.tar.gz
+        10-missing-includes.patch
+        20-backtrace-guards.patch"
```

Comment:
are those patches submitted upstream?

Reply:
Not yet, but I can make it happen.

Reply:
Refreshing my memory now, the first patch is already in master, so I just need to coordinate a new release. The second is somewhat specific to this effort to build on Alpine, so I'll need to address that separately, taking your suggested compile test into consideration.